ibis/expr/operations.py

```diff
@@ -516,6 +521,65 @@ class Log10(Logarithm):
     """Logarithm base 10"""
 
 
+class Degrees(UnaryOp):
+    """Converts radians to degrees"""
+    arg = Arg(rlz.floating)
+    output_type = rlz.shape_like('arg', dt.float)
+
+
+class Radians(UnaryOp):
+    """Converts radians to degrees"""
+    arg = Arg(rlz.floating)
+    output_type = rlz.shape_like('arg', dt.float)
+
+
+# TRIGONOMETRIC OPERATIONS
+
+class TrigonometricUnary(UnaryOp):
+    """Trigonometric base unary"""
+    arg = Arg(rlz.numeric)
+    output_type = rlz.shape_like('arg', 'float')
+
+
+class TrigonometricBinary(BinaryOp):
+    """Trigonometric base binary"""
+    left = Arg(rlz.numeric)
+    right = Arg(rlz.numeric)
+    output_type = rlz.shape_like('left', 'float')
```

Comment:
I think if any of the operands are columnar then we need to return columnar result too. 
To achieve that use:

```python
output_type = rlz.shape_like('args', dt.float64)
```